src/background/icon-manager.ts

```diff
@@ -1,4 +1,6 @@
+import { FilterMode } from 'generators/css-filter';
```

Comment:
Please use relative file paths (`../`). Please remove spaces between brackets according to our code style (`npm run code-style`).